warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -177,27 +178,29 @@ def go_forward(self, browser_instance=None):
             self.current_browser.forward()
 
     def check_url(self, url):
-        """To check whether the user provided url is valid or not."""
+        """To check whether the user provided url is valid or not.
+           Returns:
+                    1. status(bool)= True / False.(Whether the url can be reached)
+                    2. url : The actual url itself
```

Comment:
Hi @brettleehari,

Please add documentation here that clearly states states that this function would internally "open" a URL to validate it.

Thanks,
Sanika

Reply:
Done! Explicitly added a note